lib/glimesh_web/templates/gct/lookup_user.html.eex

```diff
@@ -0,0 +1,166 @@
+                    <div class="col-sm-6">
+                        <%= label :user, gettext("Twitter") %>
+                        <%= text_input :user, :twitter, [value: @user.social_twitter, class: "form-control", disabled: true] %>
+                    </div>
+                    <div class="col-sm-6">
+                        <%= label :user, gettext("Youtube") %>
+                        <%= text_input :user, :youtube, [value: @user.social_youtube, class: "form-control", disabled: true] %>
+                    </div>
+                </div>
+                <!-- Row end -->
+                <!-- Row start -->
+                <div class="row mt-1">
+                    <div class="col-sm-6">
+                        <%= label :user, gettext("Discord") %>
+                        <%= text_input :user, :discord, [value: @user.social_discord, class: "form-control", disabled: true] %>
+                    </div>
+                    <div class="col-sm-6">
+                        <%= label :user, gettext("Instagram") %>
+                        <%= text_input :user, :instagram, [value: @user.social_instagram, class: "form-control", disabled: true] %>
+                    </div>
+                </div>
+                <!-- Row end -->
+                <!-- Row start -->
+                <div class="row mt-1">
+                    <div class="col-sm-6">
+                        <%= label :user, gettext("Youtube Teaser URL") %>
+                        <%= text_input :user, :email, [value: @user.youtube_intro_url, class: "form-control", disabled: true] %>
+                    </div>
+                    <div class="col-sm-6">
+                        <%= label :user, gettext("Locale") %>
+                        <%= text_input :user, :locale, [value: @user.locale, class: "form-control", disabled: true] %>
+                    </div>
+                </div>
+                <!-- Row end -->
+                <!-- Row start -->
+                <div class="row mt-1">
+                    <div class="col">
+                        <%= label :user, gettext("Profile Content") %>
+                        <%= textarea :user, :profile_content_md, [value: @user.profile_content_md ,class: "form-control mb-4", id: "textarea_edit", rows: 7, disabled: true]  %>
+                    </div>
+                </div>
+                <!-- Row end -->
+                <%= if @user.ban_reason do %>
+                <!-- Row start -->
+                <div class="row mt-1">
+                    <div class="col">
+                        <%= label :user, gettext("Ban Reason") %>
+                        <%= textarea :user, :ban_reason, [value: @user.ban_reason,class: "form-control mb-4", id: "textarea_edit", rows: 5, disabled: true]  %>
+                    </div>
+                </div>
+                <!-- Row end -->
+                <% end %>
+            </div>
+        </div>
+        </div>
+    </div>
+    <%= if @view_billing? do %>
```

Comment:
These don't seem to be in sync with what we show the user. Had some issues showing payout history, would be better to reuse the existing view for this.

Reply:
Could use same as above.

Reply:
I've temporarily disabled the payments/payouts history till I get the existing views working(most likely the next PR after this is made).